.github/workflows/tests.yaml

```diff
@@ -45,3 +45,16 @@ jobs:
       # Then uninstall only kubemarine to avoid ambiguity and to surely run pylint on sources.
       - run: pip install pylint . && pip uninstall -y kubemarine
       - run: pylint kubemarine test --disable fixme || exit 0
+  radon:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+        # Use `radon cc {paths} -a` locally for full report per function
+        # xenon checks, if radon absolute result is A
+      - run: |
+          pip install radon xenon
```

Comment:
Lets run `pip install` and radon/xenon in separate steps to split logs output for them

Reply:
Done